trunion/validators.py

```diff
@@ -45,13 +46,17 @@ def valid_receipt(request):
     if receipt['iss'] not in request.registry.settings['trunion.permitted_issuers']:
         raise HTTPConflict("Bad issuer: \"%s\"" % receipt['iss'])
```

Comment:
it would be really great to have some logging of what the actual values are so we can debug the errors more easily. Alternatively, the exception messages could contain the timestamps that it is comparing. Clock skew happens.